lib/webapp.js

```diff
@@ -30,7 +30,12 @@ module.exports = {
     api.get(config, 'projects', function (err, repos) {
       if (err) return done(err);
 
-      repos = JSON.parse(repos);
+      if(repos === null || repos === undefined) {
```

Comment:
Yes, good. Should I amend the commit and create a new pull request, or add in a new commit to change that line?